proto/ls_state.proto

```diff
@@ -30,3 +30,10 @@ message LinRegUniLSState {
   Vector regression_coeffs = 1; // regression coefficients
   double var = 2; // variance of the noise
 }
+
```

Comment:
MFA is not parametrized through location-scale, do we still add the corresponding proto state here?

Reply:
Leave it as it is for the moment, we will deal with polishing stuff at the end!